sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/literals.scala

```diff
@@ -409,6 +409,7 @@ case class Literal (value: Any, dataType: DataType) extends LeafExpression {
         DateTimeUtils.getZoneId(SQLConf.get.sessionLocalTimeZone))
       s"TIMESTAMP('${formatter.format(v)}')"
     case (v: Array[Byte], BinaryType) => s"X'${DatatypeConverter.printHexBinary(v)}'"
+    case (v: CalendarInterval, CalendarIntervalType) => IntervalUtils.toMultiUnitsString(v)
```

Comment:
Sorry if this is already asked above but why we didn't change this?

Reply:
We have not supported to parse interval from iso/SQL standard format yet

Reply:
Why did we not support iso/SQL standard format here together?

Reply:
https://github.com/apache/spark/pull/26418#discussion_r343531257